test/rt-cyclicdeadline

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+. common
+
+name=$(basename "$0")
+test_init
+create_log_file
+
+printc "YELLOW" "[INFO] Standard error being directed to: $stderr_log"
+
+output=$(cyclicdeadline --quiet --interval=3500 --duration=1s --threads=$(nproc) 2>$stderr_log)
+
+[ $? -ne 0 ] && test_failed
+
+printc "YELLOW" "[WARN] stderr:"
+cat $stderr_log
+
+printc "NC" "[INFO] Command executed successfully"
+
+# Check program output behavior
+final_table='T:\s*\d+\s*\(\s*\d+\s*\)\s*I:\d+\s*C:\s*\d+\s*Min:\s*\d+\s*Act:\s*\d+\s*Avg:\s*\d+\s*Max:\s*\d+'
+app_regex=(
+	'Using all CPUS' \
+	'interval: 2100:3500' \
+	'deadline thread \d+' \
+	'thread\[\d+\]\s*runtime=\d+us\s*deadline=\d+us' \
+	$final_table
+	)
```

Comment:
For the record, these are very fragile. The application's output or its output style may change from one version to the next, making it difficult to maintain tests.